doc/releasenotes/15_0_0_summary.md

```diff
@@ -115,14 +115,17 @@ The following VTTablet flags were deprecated in 7.0. They have now been deleted
 - --pool-name-prefix
 - --enable-autocommit Autocommit is always allowed
 
-#### <a id="vttablet-startup-flag-deprecations"/>vttablet startup flag deprecations
```

Comment:
i think this accidentally undoes some stuff that landed in `main` between your other PR and now

Reply:
yeah looks like it is the case.. let me first land https://github.com/vitessio/vitess/pull/11597 and then I can merge those changes to this. This will results in lets conflicts.